### PR TITLE
Add support for multi languages

### PR DESCRIPTION
**Pull request by thebergamo**
For supporting multi languages, I start working in this branch.

In this case, just translating tutorials, but this basis can be applied to any part of the hapijs.com website.

This PR are Work in Progress, just sending this for getting some ideas to improve the PR.

### CODE REVIEW

File: lib/routes.js [hunk 1]
```diff
@@ -88,22 +89,22 @@ exports.routes.push({
 
 exports.routes.push({
     method: 'GET',
-    path: '/tutorials/{tutorial?}',
+    path: '/tutorials/{lang}/{tutorial?}',
```
Comment (luanmuniz) [inline lib/routes.js]: Maybe its best `/tutorials/tutorial/lang` and default to english when the `/lang` its not avaliable. That way the links in the internet won't stop working.

Comment (nlf) [inline lib/routes.js]: great point @luanmuniz we definitely want to make sure existing links don't break. i'd be fine with using a query parameter to specify language also

Comment (thebergamo) [inline lib/routes.js]: hum... is a great idea using query parameters! Tonight I will fix this :D

**Comment by thebergamo** [comment]
@nlf at this time, I think just translating the tutorials page, but makes more sense translating the entire site, right?

**Comment by nlf** [comment]
yeah i think eventually having the whole site translated would be awesome

**Comment by nlf** [comment]
remember though that i said *eventually*, for a first iteration getting the tutorials up is a really big win and i would *love* to merge that before we bother translating other pages

**Comment by thebergamo** [comment]
@nlf Well, I think this is finished.

Following your last message, only translated the tutorials page. Can you review?

**Comment by thebergamo** [comment]
news about this?

**Comment by nlf** [comment]
Sorry @thebergamo planning to review today. Thank you for taking the time to put this together!

File: templates/index.jade [hunk 2]
```diff
@@ -178,4 +178,4 @@ block content
           em localhost:8000/hello
           |  in your browser.
 
-      a(href='/tutorials', class='button button-primary') Read more about getting started with hapi
+      a(href='/tutorials/en_US', class='button button-primary') Read more about getting started with hapi
```
Comment (nlf) [inline templates/index.jade]: the `/en_US` is unnecessary now, isn't it?

Comment (thebergamo) [inline templates/index.jade]: Oh yes! A lot of places to add/remove this, my bad.

**Comment by nlf** [comment]
@thebergamo any chance you can pull the changes from #294 in to this pull request? that way we only have one merge to get this all working.

once that's done this looks good to me :+1: thanks again for all your hard work @thebergamo and @alanleite

**Comment by thebergamo** [comment]
Well, no answers about @alanleite, I will include him updates and merge with my PR.

**Comment by alanleite** [comment]
Sorry, I'm a little busy these days... thanks for the help @thebergamo

**Comment by thebergamo** [comment]
@nlf finally!! When you'll merge, can you rebase in a sigle commit?

**Comment by thebergamo** [comment]
@nlf news about this?

**Comment by nlf** [comment]
sorry, i'll get this reviewed and hopefully merged today

**Pull request merged**